src/services/__tests__/lollipopService.test.ts

```diff
@@ -1,67 +1,27 @@
 import { RevokeAssertionRefInfo } from "@pagopa/io-functions-commons/dist/src/entities/revoke_assertion_ref_info";
-import { sha256 } from "@pagopa/io-functions-commons/dist/src/utils/crypto";
-import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
-import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
-import { ActivatedPubKey } from "../../../generated/lollipop-api/ActivatedPubKey";
-import { AssertionTypeEnum } from "../../../generated/lollipop-api/AssertionType";
-import { PubKeyStatusEnum } from "../../../generated/lollipop-api/PubKeyStatus";
-import { LollipopApiClient } from "../../clients/lollipop";
 import { base64EncodeObject } from "../../utils/messages";
-import * as E from "fp-ts/Either";
-import * as t from "io-ts";
-import {
-  aLollipopAssertion,
-  anAssertionRef,
-  anEncodedJwkPubKey
-} from "../../__mocks__/lollipop";
-import { aFiscalCode } from "../../__mocks__/user_mock";
+import { anAssertionRef } from "../../__mocks__/lollipop";
 import LollipopService from "../lollipopService";
-import { AssertionFileName } from "../../../generated/lollipop-api/AssertionFileName";
 import * as appInsights from "applicationinsights";
 
 const mockSendMessage = jest.fn();
 jest.mock("@azure/storage-queue", () => ({
   QueueClient: jest.fn().mockImplementation((_, __) => {
     return {
-      sendMessage: mockSendMessage
+      sendMessage: mockSendMessage,
     };
-  })
+  }),
 }));
 
 beforeEach(() => {
   jest.clearAllMocks();
 });
 
-const mockActivatePubKey = jest.fn();
-const mockLollipopApiClient = {
-  ping: jest.fn(),
-  activatePubKey: mockActivatePubKey,
-  generateLCParams: jest.fn(),
-  reservePubKey: jest.fn()
-} as ReturnType<LollipopApiClient>;
+const mockTelemetryClient = {
+  trackEvent: jest.fn(),
+} as unknown as appInsights.TelemetryClient;
```

Comment:
The appInsights telemetryClient is not used anymore, we can remove it

Reply:
removed in 33ceeb92